src/inline.css.ts

```diff
@@ -49,44 +45,34 @@ export const inline = style({
         [supportsFlexGap]: {
             margin: 0, // restore
             pointerEvents: 'auto', // restore
-            display: 'inline-flex',
             flexDirection: 'row',
             gap: space,
         },
     },
 });
 
-export const fullWidth = style([
-    inline,
-    {
-        display: ['flex', 'grid'],
-        // style repeated to override the inline @supports style
-        '@supports': {
-            [supportsFlexGap]: {
-                display: ['flex', 'grid'],
-            },
-        },
-    },
-]);
+export const fullWidth = style({
+    /**
+     * @deprecated - this should be "flex"
+     * Once changed, the usages that depend on this behavior should be migrated to use <Grid>
+     * See "inline-cases" private story
+     */
+    display: 'grid',
+});
 
-export const wrap = style([
-    inline,
-    {
-        flexWrap: 'wrap',
-    },
-]);
+export const wrap = style({
+    display: 'inline-flex',
+    flexWrap: 'wrap',
+});
 
-export const noFullWidth = style([
-    inline,
-    {
-        display: ['inline-flex', 'inline-grid'],
-        '@supports': {
-            [supportsFlexGap]: {
-                display: 'inline-flex',
-            },
-        },
-    },
-]);
+export const noFullWidth = style({
+    display: 'inline-grid',
```

Comment:
Same thing that [Abel commented](https://github.com/Telefonica/mistica-web/pull/1007/files#r1470741091). Why removing the fallback to inline-flex if grid is not supported?

As far as I know, inline-grid is not supported in all the browsers that webapp supports (https://caniuse.com/?search=inline-grid)

Reply:
added fallback